x/cdp/types/msg.go

```diff
@@ -45,9 +37,15 @@ func (msg MsgCreateCDP) Type() string { return "create_cdp" }
 
 // ValidateBasic does a simple validation check that doesn't require access to any other information.
 func (msg MsgCreateCDP) ValidateBasic() error {
-	if msg.Sender.Empty() {
+	if msg.Sender == "" {
```

Comment:
fyi, i found that `AccAddressFromBech32` does the empty check so we should be okay to no include this check